.github/workflows/ci.yml

```diff
@@ -16,33 +16,17 @@ env:
 
 jobs:
   docker_image:
-    name: 'Get docker sc_tools image'
-    runs-on: ubuntu-latest
-
-    outputs:
-      sc_tools: ${{ steps.docker.outputs.sc_tools }}
-
-    steps:
-      - name: Checkout repository
-        run: |
-          git clone https://github.com/siliconcompiler/siliconcompiler
-
-      - name: Get image name
-        id: docker
-        run: |
-          pip install siliconcompiler
-          git -C siliconcompiler checkout v$(sc -version)
-          pip install -r siliconcompiler/setup/docker/requirements.txt
-          sc_tools_name=$(python3 siliconcompiler/setup/docker/builder.py --tool tools --registry ${{ env.REGISTRY }})
-          echo "sc_tools=${sc_tools_name}" >> $GITHUB_OUTPUT
+    uses: siliconcompiler/siliconcompiler/.github/workflows/docker_image.yml@reuse-docker-image
+    with:
+      tool: 'tools'
```

Comment:
this needs the sc-version corrected.
```
pip install siliconcompiler
git -C siliconcompiler checkout v$(sc -version)
```

You probably need to add a pre-step to get this and use it

Reply:
Hm yes that doesn't seem super pretty to me.
What do you think of having a `release: true` input or `version: latest` to move that logic into `docker_image.yml`?

Reply:
Note to myself should be possible without install using:
`git describe --abbrev=0 --tags`

Reply:
`sc_version: latest` seems fine to me.